### ISSUE TITLE
Thank you (I applied this to GELU)

### ISSUE THREAD

**Issue by zer0int** [Wenlin-Chen/geometric-parameterization#1]
Thank you for the inspiration from your paper!
I applied this to CLIP (GELU) because "faster / fewer epochs" and "less overfit / better generalization" sounded appealing & worth a shot.

CLIP ViT-L/14 fine-tuned with .theta and .r instead of .weight has improved accuracy (on the val dataset - 10% of a COCO 40k; it's nothing much, but it's honest work for 1 GPU available!) vs. the pre-trained OpenAI/ViT-L/14. What's even more puzzling (albeit I don't know if this generalizes), it seems the fine-tuned model is much more robust to unintentional typographic attacks (text in images).

Figured I'd just leave this here, together with my gratitude. Kind regards!

[https://github.com/zer0int/CLIP-fine-tune](https://github.com/zer0int/CLIP-fine-tune)
